bundles/CoreBundle/src/EventListener/Frontend/ElementListener.php

```diff
@@ -260,6 +260,7 @@ private function handleStudioPreview(int $id, UserInterface $user): void
 
     private function getLatestVersion(int $id, UserInterface $user): ?Concrete
     {
+        /** @var Concrete|null $dataObject */
         $dataObject = Service::getElementById('object', $id);
 
         if ($dataObject === null) {
```

Comment:
I would change the if and remove the phpdoc, the getElementById() can also return Folders.
```suggestion
        if (!$dataObject instanceof Concrete) {
```

Reply:
True! Thank you for the review! Applied the change!